extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -232,19 +232,10 @@ export class LinkerManager {
       return false;
     }
 
-    const headNode = this.ampdoc_.getHeadNode();
-    const linkerCreatedEl =
-      headNode instanceof ShadowRoot
-        ? this.ampdoc_.getBody()
-        : headNode.querySelector(
-            'meta[name="amp-google-client-id-api"][content="googleanalytics"]'
-          );
-
-    if (linkerCreatedEl.hasAttribute(LINKER_CREATED)) {
+    if (!this.ampdoc_.registerService(LINKER_CREATED)) {
```

Comment:
Nit: Just `return this.ampdoc_.registerService(LINKER_CREATED)`?

Reply:
👍 Good point